composer.json

```diff
@@ -21,12 +21,12 @@
         "issues": "https://github.com/zumba/mongounit/issues"
     },
     "require": {
-        "php": ">=5.5.0",
-        "ext-mongodb": "^1.5.0",
-        "mongodb/mongodb": ">=1.1.1 <1.5"
```

Comment:
@nhatthm Why did you have to drop the support for the older mongo drivers? That will block people from using this lib with older versions of MongoDB server and extension.

Reply:
@jrbasso https://packagist.org/packages/mongodb/mongodb v1.5 doesn't support mongodb extension older than 1.6

Reply:
Sounds good. We are going to bump the major version so people can still use the lib without upgrading the extension in their end.

Thanks for your contribution. 👍